keyboards/ergodox_ez/keymaps/danielo515/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+  Set any config.h overrides for your specific keymap here.
+  See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+*/
+#define IGNORE_MOD_TAP_INTERRUPT
```

Comment:
This needs an include guard, or it can lead to cases where it's redefining settings. 
```suggestion
#pragma once 

#define IGNORE_MOD_TAP_INTERRUPT
```